extensions/podman/src/extension.ts

```diff
@@ -208,26 +216,60 @@ export async function checkDefaultMachine(machines: MachineJSON[]): Promise<void
   // As a last check, let's see if the machine that is running is set by default or not on the CLI.
   // if it isn't, we should prompt the user to set it as default, or else podman CLI commands will not work
   const runningMachine = machines.find(machine => machine.Running);
-  const defaultMachine = machines.find(machine => machine.Default);
+  let defaultMachine = machines.find(machine => machine.Default);
+  if (!defaultMachine) {
+    const defaultConnection = await getDefaultConnection();
+    let defaultConnectionName = defaultConnection?.Name;
+    if (defaultConnectionName.endsWith('-root')) {
```

Comment:
`-root` is used multiple times in the code, should it be a const variable in case we want to change it?

Reply:
We should add a comment that the reason that it's `-root` is because we're updating it to be that way?

Reply:
Done 👍